nimbro_topic_transport/launch/tcp_sender.launch

```diff
@@ -1,5 +1,6 @@
 <launch>
 	<arg name="target" default="localhost" />
+	<arg name="allow_bidirectional" default="true" />
```

Comment:
I would prefer it if we could keep the bidirectional launch files completely separate from the standard ones. The launch files were intended as a "simple-as-possible" configuration.

If you separate them, it would suffice to have one launch file called "bidirectional_1.launch" and "bidirectional_2.launch" for the two machines, each with one sender and one receiver node inside. That would make things clearer than the "_back" suffix.